### DISCUSSION TITLE
Blank page break in PDF/DOCX export output

### DISCUSSION THREAD

**Discussion by kabusko** [eclipse-birt/birt#992] [Q&A]
Before saying this is an issue, I want to know what the community thinks about.

We are using the latest version of BIRT (3.9.0). 

Our report uses the master/detail to display the tree-based structure  of our data. For this we use tables embedded in row. For every master element row, a table is used to display each of its children. 

For example, if a master element has 2 children, we will have two tables embedded in its row with each presenting details of each  child. 

Now the issue (or bad behavior) we are facing is that these embedded tables are spaced by blank page in PDF output. We observed the same behavior with DOCX too. Even if we used "Page Break = Auto" for before/after of the tables, the same bad output comes. 

At this point, we think the pdf emitter may have an issue. But....

So based on this description, can someone recognize encountering this experience with pdf/docx export?
Do you think we are missing to properly set some properties apart from the page break properties? Which properties?

If any extra details are needed (report design, etc.) in order to understand further this case, please let me know. I will do my best to provide the information.

Thanks,

**Comment by hvbtup** [comment]
Please try to create a self-contained example report demonstrating the issue (one which works with the ClassicModels builtin example database or with sample data from a file). Create a ZIP archive of *.rptdesign and data file and attach it here.
